### ISSUE TITLE
Add drop down for selecting offical Bevy examples

### ISSUE THREAD

**Issue by LiamGallagher737** [LiamGallagher737/learnbevy#12]
Add useful addition would be a drop down to select from official example like on https://go.dev/play/. 

![image](https://github.com/LiamGallagher737/learnbevy/assets/86995276/049510ca-ec15-4abd-9972-a1d48a86d40b)

### Implementation

This would depend on #5 adding all the same assets the Bevy examples use.

#### Ui

Using [shadcn's combobox](https://www.shadcn-svelte.com/docs/components/combobox) would be the best fit due to having a search box and there being a lot of examples.

#### Getting examples

Using the GitHub api to [get repo contents](https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28#get-repository-content) and then recursively finding all the examples seems like the easiest way to me without making it to manual. 

Getting the examples themselves would just be a case of filling in the following url.

https://raw.githubusercontent.com/bevyengine/bevy/release-[VERSION]/examples/[EXAMPLE_PATH]

**Comment by LiamGallagher737** [comment]
Partially completed in #16, examples for main branch will be in separate PR

**Comment by LiamGallagher737** [comment]
Should also add the ability to load an example from query string

**Comment by LiamGallagher737** [comment]
Completed with #21
